app/helpers/social_image_helper.rb

```diff
@@ -0,0 +1,55 @@
+module SocialImageHelper
+  # After this date we use SocialPreview controller directly rather than passing to URL2PNG.
+  # Keeping old URLs around since they are already generated.
+  SOCIAL_PREVIEW_MIGRATION_DATETIME = DateTime.new(2019, 3, 24)
```

Comment:
I feel this should be set explicitly for future proofing. `DateTime.new` convienently ignores the current `Time.zone` defaulting to UTC which is fine (because we want UTC midnight for the 24th of March 2019) but istinctively a dev could ask themselves in which time zone this date is. 

My suggestion is to switch to: `Time.zone.parse("2019-03-24T00:00:00Z")` and add a test for it, which would fail if this date is not exactly "2019-03-24T00:00:00Z" because this date needs to be set in stone for ever and ever :D

Reply:
Great suggestion. Updated the constant. Thank you!